components/SummaryStats/SummaryStats.tsx

```diff
@@ -35,7 +55,7 @@ export default function SummaryStats() {
   const rows = () => {
     return Object.entries(config).map(([key, entry]) => {
```

Comment:
Doesn't matter because it's just a variable name, but technically the `entry` is really the whole `[key, value]` tuple.

Reply:
truuue. i will fix that the next time i can.

(it does matter!)